lua/ulib/server/ucl.lua

```diff
@@ -414,6 +415,7 @@ function ucl.groupAllow( name, access, revoke )
 
 		ucl.saveGroups()
 
+		hook.Call( ULib.HOOK_GROUP_ACCESS_CHANGE, _, name, access, revoke or false )
```

Comment:
I see the "abc or false" pattern a lot in your proposed changes. Did you do this so that the value was forced to be true/false rather than true/false/nil? Because it's not strictly necessary if the passed value is only being used for conditionals.